lib/ex_plasma/client.ex

```diff
@@ -38,7 +37,7 @@ defmodule ExPlasma.Client do
     |> Deposit.new()
     |> Client.deposit()
   """
-  @spec deposit(Deposit.t() | binary(), map() | list()) :: tuple()
+  @spec deposit(Transaction.t() | binary(), map() | list()) :: tuple()
   def deposit(tx_bytes, options \\ %{})
 
   def deposit(tx_bytes, options) when is_list(options),
```

Comment:
why allow options as lists?

Reply:
Removed. was playing around with a nicer API via keyword list, but not really an important need 😄